Solutions/Marain.ContentManagement.Api.Host/Marain/Cms/Api/Host/Startup.cs

```diff
@@ -0,0 +1,46 @@
+﻿// <copyright file="Startup.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+[assembly: Microsoft.Azure.WebJobs.Hosting.WebJobsStartup(typeof(Marain.Cms.Api.Host.Startup))]
+
+namespace Marain.Cms.Api.Host
+{
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Hosting;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Logging;
+
+    /// <summary>
+    /// Startup code for the Function.
+    /// </summary>
+    public class Startup : IWebJobsStartup
+    {
+        /// <inheritdoc/>
+        public void Configure(IWebJobsBuilder builder)
+        {
+            IServiceCollection services = builder.Services;
+
+            IConfigurationRoot root = Configure(services);
+            services.AddLogging(builder =>
+            {
+                builder.AddConsole();
+                builder.SetMinimumLevel(LogLevel.Debug);
+            });
+
+            services.AddTenantedContentManagementApi(root, config => config.Documents.AddSwaggerEndpoint());
+        }
+
+        private static IConfigurationRoot Configure(IServiceCollection services)
+        {
+            IConfigurationBuilder configurationBuilder = new ConfigurationBuilder()
```

Comment:
I have no idea what we are supposed to be doing with configuration now. Is this it?

Reply:
No, but I'll create a separate issue as we will need the various Corvus libraries to be fixed before we can get rid of it.

Reply:
Created issue #37 to track.